dub.sdl

```diff
@@ -43,9 +43,9 @@ subPackage {
   sourceFiles \
     "src/dmd/astbase.d" \
     "src/dmd/parse.d" \
-    "src/dmd/transitivevisitor.d" \
-    "src/dmd/permissivevisitor.d" \
-    "src/dmd/strictvisitor.d"
+    "src/dmd/visitors/transitive.d" \
+    "src/dmd/visitors/permissive.d" \
+    "src/dmd/visitors/strict.d"
```

Comment:
Add `"src/dmd/visitors/package.d"`

Reply:
And also `parsetime`.

Reply:
The reason why it didn't break, is because this package isn't used anymore. There's `frontend` now...